deploy/efi-installer-node-amd64.conf

```diff
@@ -0,0 +1,68 @@
+# vim: syntax=sh
+
+DEPLOYMENT_TYPE=installer
+ARCH=amd64
+TARGET_HOSTNAME=molior-node
+INSTALLER_BOOTTYPE=efi
+INSTALL_PACKAGE=molior-node-$ARCH-efi
+LINUX_CMDLINE="biosdevname=0 net.ifnames=0"
+
+INSTALLER_ASK_HOSTNAME=no
+INSTALLER_ASK_CONFIRM=no
+INSTALLER_ASK_REBOOT=no
+
+# do not install man pages, /usr/share/doc, ...
+MINIMIZE=1
+
+SFDISK="unit: sectors
+label: gpt
+efi:  type=C12A7328-F81F-11D2-BA4B-00A0C93EC93B, size=96M, start=2048
+swap: type=0657FD6D-A4AB-43C4-84E5-0933C84B4F4F, size=512M,
+root: type=0FC63DAF-8483-4772-8E79-3D69D8477DE4"
+
+PART1_FS=vfat
+PART1_MNT=/boot/efi
+PART2_FS=swap
+PART3_FS=ext4
+PART3_MNT=/
+
+preinst_deployment_installer()
+{
+  # FIXME: do not overwrite sourced preinst function
+  cp -ar $target $WORK_DIR/instroot
+
+  if [ "$SUITE" = "stretch" ]; then
+    echo " * configuring stretch-backports"
+    cat >$target/etc/apt/preferences.d/stretch-backports <<EOF
+Package: *
+Pin: release a=stretch-backports
+Pin-Priority: 900
+
+Package: *
+Pin: release a=stretch
+Pin-Priority: 800
+EOF
+  fi
+}
+
+postinst_deployment_installer()
+{
+  # use public ATP sources
+  cat >$target/etc/apt/sources.list << EOF
+# APT Sources for project molior 1.3
+# GPG-Key: http://molior.info/archive-keyring.asc
+
+deb http://deb.debian.org/debian $SUITE main
+EOF
+
+  if [ "$SUITE" = "stretch" ]; then
+    cat >$target/etc/apt/sources.list << EOF
```

Comment:
overwrite or append?

Reply:
overwrite, since we want to replace the internal apt sources